.storybook/preview.js

```diff
@@ -53,4 +53,26 @@ addDecorator(Story => (
   </div>
 ));
 
-addDecorator(withKnobs);
\ No newline at end of file
+addDecorator(withKnobs);
+
+addParameters({
+  options: {
+    storySort: (a, b) => {
+      const introRegex = /^Basics\|/;
+      // We want the Basic story at the top
+      if (introRegex.test(a[1].kind)) {
+        // Make sure Introduction is at the top
+        if (b[1].kind.match(/|Introduction$/) && introRegex.test(a[1].kind)) return 1;
+        return -1
+      } else if (introRegex.test(b[1].kind)) {
+        return 1
+      }
+
+      // Sort the other stories by ID
+      // https://github.com/storybookjs/storybook/issues/548#issuecomment-530305279
+      return a[1].kind === b[1].kind
+        ? 0
+        : a[1].id.localeCompare(b[1].id, { numeric: true })
+    },
```

Comment:
Is this necessary? Were there issues with the previous sort?

Reply:
Yes it wasn't working correctly. Not sure if it was in relation to babel or what, but the imports weren't being consistently sorted.

I believe this will be resolved with v6 migration and would prefer to not worry about it since it doesn't need to be changd.

Reply:
Ah I see what I missed. I forgot to add the Introduction import first